src/index.tsx

```diff
@@ -74,10 +74,10 @@ function CustomApp() {
     useEffect(() => {
         if (localStorage.getItem('pass') === PASSWORD_HASH) return setAuth(true)
 
-        const passMatch = window.location.search.match(/\bpass=\w+,\b/)
+        const passMatch = window.location.search.match(/pass=(.+?)($|&)/)
         if (passMatch && passMatch[0]) {
```

Comment:
when you use brackets () you get extra matching groups
so passMatch[1] will give you the password